### ISSUE TITLE
Log-in form on SanPark skin doesn't work

### ISSUE THREAD

**Issue by mbjones** [NCEAS/metacat#1016]
---

Author Name: **Jing Tao** (Jing Tao)
Original Redmine Issue: 6427, https://projects.ecoinformatics.org/ecoinfo/issues/6427
Original Date: 2014-02-26
Original Assignee: Jing Tao

---

If you click the login button on the Sanpark skin even though input a valid user/password, the login doesn't go through. The browser's url text box will show a metacat login servlet url even with the password!

**Comment by mbjones** [comment]
---

Original Redmine Comment
Author Name: **Jing Tao** (Jing Tao)
Original Date: 2014-02-26T19:08:02Z

---

But the SAEON skin works.

**Comment by mbjones** [comment]
---

Original Redmine Comment
Author Name: **Jing Tao** (Jing Tao)
Original Date: 2014-02-27T06:14:08Z

---

I dug around and found the log-in worked on Firefox but didn't work on Chrome. The issue is the Ajax.Updater didn't send the request to Metacat at all.

I discussed with Ben and he suggested to add "return false" in the attribute "onsubmit" in the loginForm in the sanparksLogin.jsp. Then it works. I also add a new attribute "method="post"" in that form as well. This fixed the issue to put the parameters to the url. Those changed was applied to the logoutForm as well.

Then I found another issue that the re-login didn't work after the user logged out. It turned out that the re-login form is generated from the sanparksLogin.xsl. We should add above things to the sanparksLogin.xsl as well.
